requirements.txt

```diff
@@ -11,4 +11,4 @@ peewee==2.6.4
 python-dateutil==2.4.2
 django-dbbackup==2.3.2
 ifcfg==0.9.3
-requests==1.1.0
+requests==2.10.0
```

Comment:
There's a version `2.11.1` on [PyPi](https://pypi.python.org/pypi/requests/) so maybe you can use that instead?